### PR TITLE
Home-Page Subscription Form Upgrades

### PR DESCRIPTION
**Pull request by jperegrino75**
Made the following changes to subscription form on audioverse.org Home page:

1. Added new verbiage to the Newsletter form
2. added two checkboxes: newsLetter and Weekly pics
3. integrated to Active Campaign

https://audioverse7.atlassian.net/browse/AV-940

### CODE REVIEW

File: src/components/molecules/newsletterForm.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+						value={email}
+						setValue={setEmail}
+					/>
+				</div>
+				<div>
+					<Checkbox
+						label={
+							<FormattedMessage
+								id="newsLetterForm__checkboxNewsletter"
+								defaultMessage="Newsletter typically about once a month"
```
Comment (alangumer) [inline src/components/molecules/newsletterForm.tsx:85]: Use parentheses or another separator to specify the frequency.

File: src/components/molecules/newsletterForm.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+						checked={checkboxNewsLetter}
+						toggleChecked={() => setCheckboxNewsLetter((v) => !v)}
+						value="2"
+						name="nlbox[]"
+					/>
+					<Checkbox
+						label={
+							<FormattedMessage
+								id="newsLetterForm__checkboxWeeklyPics"
+								defaultMessage="Weekly Picks goes out every Monday"
```
Comment (alangumer) [inline src/components/molecules/newsletterForm.tsx:97]: Same here.

**Review by alangumer — CHANGES_REQUESTED** [review]
Great job, just fix the small things that I mentioned.

File: src/components/molecules/newsletterForm.tsx (hunk 1 above)
Comment (jperegrino75) [reply src/components/molecules/newsletterForm.tsx:85]: done

**Review by alangumer — APPROVED** [review]

**Pull request merged**